From 76f2ca9318a5eb8157cec187246762407287afab Mon Sep 17 00:00:00 2001
From: Robert Kanter <rkanter@cloudera.com>
Date: Mon, 14 Jan 2013 11:55:19 -0800
Subject: [PATCH 1250/1357] MR1: CLOUDERA-BUILD. mapreduce.FileOutputCommitter#abortTask should throw IOException (CDH-9238)

---
 .../mapreduce/lib/output/FileOutputCommitter.java  |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/mapred/org/apache/hadoop/mapreduce/lib/output/FileOutputCommitter.java b/src/mapred/org/apache/hadoop/mapreduce/lib/output/FileOutputCommitter.java
index f3334f1..5ac1cec 100644
--- a/src/mapred/org/apache/hadoop/mapreduce/lib/output/FileOutputCommitter.java
+++ b/src/mapred/org/apache/hadoop/mapreduce/lib/output/FileOutputCommitter.java
@@ -221,7 +221,7 @@ public class FileOutputCommitter extends OutputCommitter {
    * Delete the work directory
    */
   @Override
-  public void abortTask(TaskAttemptContext context) {
+  public void abortTask(TaskAttemptContext context) throws IOException{
     try {
       if (workPath != null) { 
         context.progress();
-- 
1.7.0.4

