From b435e84dc3adadb7f8ccdd9281643a151760b5c6 Mon Sep 17 00:00:00 2001
From: Roman Shaposhnik <rvs@cloudera.com>
Date: Tue, 27 Mar 2012 12:03:39 -0700
Subject: [PATCH 1130/1357] MR1: CLOUDERA-BUILD. Seed the classpath of MR1 with MR2 value (when available)

---
 bin/hadoop |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/bin/hadoop b/bin/hadoop
index 61409b7..d5abc35 100755
--- a/bin/hadoop
+++ b/bin/hadoop
@@ -176,6 +176,11 @@ for f in $HADOOP_HOME/build/hadoop-tools-*.jar; do
   TOOL_PATH=${TOOL_PATH}:$f;
 done
 
+HADOOP_LIBEXEC_DIR=${HADOOP_LIBEXEC_DIR:-"$bin"/../libexec}
+if [ -e "${HADOOP_LIBEXEC_DIR}/hadoop-config.sh" ]; then
+  CLASSPATH="${CLASSPATH}:`. "${HADOOP_LIBEXEC_DIR}/hadoop-config.sh" ; echo $CLASSPATH`"
+fi
+
 # add user-specified CLASSPATH last
 if [ "$HADOOP_USER_CLASSPATH_FIRST" = "" ] && [ "$HADOOP_CLASSPATH" != "" ]; then
   CLASSPATH=${CLASSPATH}:${HADOOP_CLASSPATH}
-- 
1.7.0.4

