From 811f240349e85130942d2acecd5c103a5c1dacc9 Mon Sep 17 00:00:00 2001
From: Mark Grover <grover.markgrover@gmail.com>
Date: Wed, 19 Dec 2012 17:56:04 -0800
Subject: [PATCH 1233/1357] MR1: CDH-8545: Package Job Tracker High Availability

---
 bin/hadoop |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/bin/hadoop b/bin/hadoop
index ad45579..167a890 100755
--- a/bin/hadoop
+++ b/bin/hadoop
@@ -73,7 +73,7 @@ if [ $# = 0 ]; then
   echo "  balancer             run a cluster balancing utility"
   echo "  fetchdt              fetch a delegation token from the NameNode"
   echo "  jobtracker           run the MapReduce job Tracker node" 
-  echo "  jobtrackerhadaemon   run the Job Tracker HA daemon"
+  echo "  jobtrackerha         run the Job Tracker HA daemon"
   echo "  mrhaadmin            run a MapReduce HA admin client" 
   echo "  mrzkfc               run the MapReduce ZK Failover Controller daemon" 
   echo "  pipes                run a Pipes job"
@@ -244,7 +244,7 @@ elif [ "$COMMAND" = "fetchdt" ] ; then
 elif [ "$COMMAND" = "jobtracker" ] ; then
   CLASS=org.apache.hadoop.mapred.JobTracker
   HADOOP_OPTS="$HADOOP_OPTS $HADOOP_JOBTRACKER_OPTS"
-elif [ "$COMMAND" = "jobtrackerhadaemon" ] ; then
+elif [ "$COMMAND" = "jobtrackerha" ] ; then
   CLASS=org.apache.hadoop.mapred.JobTrackerHADaemon
   HADOOP_OPTS="$HADOOP_OPTS $HADOOP_JOBTRACKER_OPTS"
 elif [ "$COMMAND" = "mrhaadmin" ] ; then
@@ -377,7 +377,7 @@ HADOOP_OPTS="$HADOOP_OPTS -Dhadoop.policy.file=$HADOOP_POLICYFILE"
 #           Uses su in order to assume the identity of the daemon user.
 #  "normal" - supported only when already running as the target user.
 ###########################################################################
-if [[ "$COMMAND" == *node ]] || [[ "$COMMAND" == *tracker ]]; then
+if [[ "$COMMAND" == *node ]] || [[ "$COMMAND" == *tracker ]] || [[ "$COMMAND" == mrzkfc ]] || [[ "$COMMAND" == jobtrackerha ]]; then
   command_uc=$(echo $COMMAND| tr a-z A-Z)
   user_var="HADOOP_${command_uc}_USER"
   _HADOOP_DAEMON_USER=$(eval "echo \$$user_var")
-- 
1.7.0.4

