From 5ab1a1731b97dcdc4877960cb955ae74a9bbaa24 Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@cloudera.com>
Date: Thu, 9 May 2013 15:38:10 -0700
Subject: [PATCH 1313/1357] MR1: CLOUDERA-BUILD. CDH-9235. Fix API incompatibility in mapreduce.lib.jobcontrol.JobControl

---
 .../mapreduce/lib/jobcontrol/JobControl.java       |   11 ++++++-----
 1 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/lib/jobcontrol/JobControl.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/lib/jobcontrol/JobControl.java
index 4ec384f..aaf598c 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/lib/jobcontrol/JobControl.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/lib/jobcontrol/JobControl.java
@@ -22,6 +22,7 @@ import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Hashtable;
 import java.util.Map;
+import java.util.List;
 
 import org.apache.hadoop.mapreduce.lib.jobcontrol.ControlledJob.State;
 
@@ -87,32 +88,32 @@ public class JobControl implements Runnable {
   /**
    * @return the jobs in the waiting state
    */
-  public ArrayList<ControlledJob> getWaitingJobList() {
+  public List<ControlledJob> getWaitingJobList() {
     return toArrayList(this.waitingJobs);
   }
 	
   /**
    * @return the jobs in the running state
    */
-  public ArrayList<ControlledJob> getRunningJobList() {
+  public List<ControlledJob> getRunningJobList() {
     return toArrayList(this.runningJobs);
   }
 	
   /**
    * @return the jobs in the ready state
    */
-  public ArrayList<ControlledJob> getReadyJobsList() {
+  public List<ControlledJob> getReadyJobsList() {
     return toArrayList(this.readyJobs);
   }
 	
   /**
    * @return the jobs in the success state
    */
-  public ArrayList<ControlledJob> getSuccessfulJobList() {
+  public List<ControlledJob> getSuccessfulJobList() {
     return toArrayList(this.successfulJobs);
   }
 	
-  public ArrayList<ControlledJob> getFailedJobList() {
+  public List<ControlledJob> getFailedJobList() {
     return toArrayList(this.failedJobs);
   }
 	
-- 
1.7.0.4

